webpack.config.ts

```diff
@@ -1,12 +1,14 @@
-'use strict';
-const path = require('path');
+import path from 'path';
+import CopyWebpackPlugin from 'copy-webpack-plugin';
+import TerserPlugin from 'terser-webpack-plugin';
+import MiniCssExtractPlugin from 'mini-css-extract-plugin';
+import webpack from 'webpack';
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports,@typescript-eslint/no-var-requires
```

Comment:
@bfred-it, I added the disabled lint rules here. Other options are adding a `declare module 'size-plugin'` in globals.d.ts or a PR to the https://github.com/GoogleChromeLabs/size-plugin. I can definitely send a PR over there at some point, but just have some other things on the go at the moment.

Reply:
> I can definitely send a PR over there at some point, but just have some other things on the go at the moment.

👍

Reply:
I'd rather have `declare module 'size-plugin'` in globals.d.ts than a comment here + `require` but it doesn't seem to work

Reply:
I'll make the change and try to get to creating the types for that package in the next week or so, so that we can remove the temp `declare module 'size-plugin'`.

Reply:
We can do without the TODO, it’s not really something we **need** to type. Once the types are in it’ll just work